Dockerfile

```diff
@@ -64,4 +69,4 @@ USER pptruser
 WORKDIR /home/pptruser/app
 
 ENTRYPOINT ["dumb-init", "--"]
-CMD ["/bin/archive"]
+CMD ["/bin/archivebox"]
```

Comment:
Can you leave this as `/bin/archive` for now to stay consistent with the rest of the docs.